cmake/Gems.cmake

```diff
@@ -71,6 +71,8 @@ function(ly_create_alias)
         ly_add_dependencies(${ly_create_alias_NAME} ${final_targets})
         # copy over all the dependent target interface properties to the alias
         o3de_copy_targets_usage_requirements(TARGET ${ly_create_alias_NAME} SOURCE_TARGETS ${final_targets})
+        # Register the targets this alias aliases
+        set_property(GLOBAL APPEND PROPERTY LY_ALIASED_TARGETS_${ly_create_alias_NAME} ${final_targets})
```

Comment:
Can you make this property `O3DE_ALIASED_TARGETS`?
We are moving away from making new cmake variables that have LY as part of its name

Reply:
Fixed 👍 

I will open up another PR to refactor all TIAF cmake scripts to reflect the move away from `ly` to `o3de`.